core/db/migrate/20210114182625_create_spree_payment_methods_stores.rb

```diff
@@ -0,0 +1,8 @@
+class CreateSpreePaymentMethodsStores < ActiveRecord::Migration[6.0]
+  def change
+    create_table :spree_payment_methods_stores, id: false do |t|
+      t.belongs_to :payment_method
```

Comment:
@MatthewKennedy will this automatically create index for that field? if not we should add it :)